### PR TITLE
add global scope validator docs

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/develop/writing-a-custom-global-scope-validator.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+     *
+     * @param authzReqMessageContext Authorization request.
+     * @return True if the user has enough permission to generate tokens or authorization codes with requested
+     * scopes or no scopes are requested, otherwise false.
+     * @throws IdentityOAuth2Exception Identity Oauth Exception.
+     */
+    Public boolean validateScope(OAuthAuthzReqMessageContext authzReqMessageContext) throws IdentityOAuth2Exception {
+
+//handle implementation 
+    Return true
```
Comment (piraveena) [inline en/docs/develop/writing-a-custom-global-scope-validator.md:40]:
```suggestion
    return true;
```

File: en/docs/develop/writing-a-custom-global-scope-validator.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    /**
+     * Validates scopes in the authorization request and manipulate the permitted scopes within the request. Engage
+     * it after application-level validators at ResponseTypeHandler level.
+     *
+     * @param authzReqMessageContext Authorization request.
+     * @return True if the user has enough permission to generate tokens or authorization codes with requested
+     * scopes or no scopes are requested, otherwise false.
+     * @throws IdentityOAuth2Exception Identity Oauth Exception.
+     */
+    Public boolean validateScope(OAuthAuthzReqMessageContext authzReqMessageContext) throws IdentityOAuth2Exception {
```
Comment (piraveena) [inline en/docs/develop/writing-a-custom-global-scope-validator.md:37]:
```suggestion
    public boolean validateScope(OAuthAuthzReqMessageContext authzReqMessageContext) throws IdentityOAuth2Exception {
```

File: en/docs/develop/writing-a-custom-global-scope-validator.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    /**
+     * Validates scopes in the token request and manipulate the permitted scopes within the request. Engage it after
+     * application-level validators at GrantHandler level.
+     *
+     * @param tokenReqMessageContext OAuthTokenReqMessageContext.
+     * @return True if the user has enough permission to generate tokens with requested scopes or
+     * no scopes are requested, otherwise false.
+     * @throws IdentityOAuth2Exception Identity Oauth Exception.
+     */
+     Public boolean validateScope(OAuthTokenReqMessageContext tokenReqMessageContext) throws IdentityOAuth2Exception {
```
Comment (piraveena) [inline en/docs/develop/writing-a-custom-global-scope-validator.md:52]:
```suggestion
     public boolean validateScope(OAuthTokenReqMessageContext tokenReqMessageContext) throws IdentityOAuth2Exception {
```

File: en/docs/develop/writing-a-custom-global-scope-validator.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+    /**
+     * Validates the scopes present in the token in the token validation flow.
+     * Engage it after application-level validators at TokenValidator level.
+     *
+     * @param tokenValidationMessageContext OAuth2TokenValidationMessageContext.
+     * @return True if scopes present in the tokens are validated successfully, otherwise false.
+     * @throws IdentityOAuth2Exception Identity Oauth Exception.
+     */
+   Public boolean validateScope(OAuth2TokenValidationMessageContext tokenValidationMessageContext)
```
Comment (piraveena) [inline en/docs/develop/writing-a-custom-global-scope-validator.md:67]:
```suggestion
   public boolean validateScope(OAuth2TokenValidationMessageContext tokenValidationMessageContext)
```

File: en/docs/develop/writing-a-custom-global-scope-validator.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+            throws IdentityOAuth2Exception {
+
+//handle implementation 
+
+ Return true;
```
Comment (piraveena) [inline en/docs/develop/writing-a-custom-global-scope-validator.md:72]:
```suggestion
 return true;
```

File: en/docs/develop/writing-a-custom-global-scope-validator.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+    /**
+     * Get the friendly name of the implemented scope validator.
+     *
+     * @return Name of the scope validator.
+     */
+   Public String getName(){
```
Comment (piraveena) [inline en/docs/develop/writing-a-custom-global-scope-validator.md:80]:
```suggestion
   public String getName(){
```

File: en/docs/develop/writing-a-custom-global-scope-validator.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+  Return “CustomGlobalScopeValidator”
```
Comment (piraveena) [inline en/docs/develop/writing-a-custom-global-scope-validator.md:81]:
```suggestion
  return “CustomGlobalScopeValidator”
```

**Review by piraveena — APPROVED** [review]

File: en/docs/develop/writing-a-custom-global-scope-validator.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+     *
+     * @return Name of the scope validator.
+     */
+   public String getName(){
+  return “CustomGlobalScopeValidator”
+}
+}
+```
+
+After you write the custom scope  validator, and register the class as an OSGI bundle, you can deploy the OSGi bundle in WSO2 IS.
```
Comment (sherenewso2) [inline en/docs/develop/writing-a-custom-global-scope-validator.md:86]: let's try to minimize the use of 'you'

After writing the custom scope validator, register the class as an OSGI bundle and deploy it in WSO2 IS.

File: en/docs/develop/writing-a-custom-global-scope-validator.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
+# Writing a Custom Global Scope Validator
+
+WSO2 Identity Server (WSO2 IS) version 5.5.0 and later support configuring scope validators for a service provider. This is another level of scope validation that can be enforced at the server level. Similar to validators engaged at application level, it has the capability to register multiple validators and execute the chain.
```
Comment (sherenewso2) [inline en/docs/develop/writing-a-custom-global-scope-validator.md:3]: another level of scope validation --> as opposed to what other level?

**Review by sherenewso2 — APPROVED** [review]

**Pull request merged**